### DISCUSSION TITLE
Lazyvim update

### DISCUSSION THREAD

**Discussion by dpetka2001** [LazyVim/LazyVim#133] [Q&A] [answered]
Has something changed with Lazyvim update since the start of release versions?
In `lua/config/lazy.lua` I have enabled `version = false`, which I understand should grab the 
latest git commit to update Lazyvim. But since the start of release versions I only get the 
updated version of the latest release? Is this how it's supposed to happen? It's as though the 
option `version = "*"` is used instead. Is there some other configuration that overrides the option 
set in `lua/config/lazy.lua` ? Or maybe I'm misunderstanding how this should work?

**Comment by folke** [comment, marked as answer]
Yes, with LazyVim, both LazyVim and lazy.nvim now default to the latest stable release.

To get the dev versions, see here: https://www.lazyvim.org/configuration/lazy.nvim

**Reply by dpetka2001** [reply]
I see thank you very much for the information.
